### DISCUSSION TITLE
Access to output text writer.

### DISCUSSION THREAD

**Discussion by Rekia-bmstu** [xunit/xunit#2424] [Question] [answered]
Hello. I've got a problem recently. I need to provide one method with test output's text writer. But I do not really understand how to access this text writer. Is is possible?

The same thing in NUnit is provided via `TestContext.Out`.

**Comment by bartelink** [comment, marked as answer]
ITestOutputHelper is the equivalent; see https://xunit.net/docs/capturing-output (Yes, it is not `static`, and hence you'll have to pass it from the Test Class ctor to where you need it)
